### PR TITLE
`lsp-iedit': add to docs, changelog, contributors

### PR DESCRIPTION
**Pull request by nbfalcon**
Add `lsp-iedit` to the docs, describing `lsp-iedit-highlights'. Add it
to the integrations section.

Add `lsp-iedit` to the changelog.

Add `lsp-iedit` to nbfalcon under CONTRIBUTORS.

### CODE REVIEW

**Comment by nbfalcon** [comment]
Docs for #2477. How do I view the generated docs, though?

**Review by ericdallo — APPROVED** [review]

**Comment by ericdallo** [comment]
@nbfalcon you can build it local with `make local-webpage`

File: docs/page/main-features.md [hunk 1]
```diff
@@ -94,6 +94,20 @@ In general the formatter settings are language server specific(e. g. `JDT LS` us
...
+
+- `lsp-iedit-highlights`: invoke `iedit` on the symbol highlights at point, all
+  of which will be edited in parallel.
+  
+  To finish and exit, press `C-g`, with the cursor on `iedit` highlight.
+  
+  If an `iedit` session is already active, `lsp-iedit-highlights` will simply
+  add the document highlights to it, without restarting it. This way, another
+  group of symbol highlights can be added to the current `iedit` session.
+
```
Comment (ericdallo) [inline docs/page/main-features.md:110]: I think this would fit better as a integration like Helm, ivy etc, check the above section: `Integrations`

**Review by ericdallo — CHANGES_REQUESTED** [review]
WDYT?

File: docs/page/main-features.md (hunk 1 above)
Comment (nbfalcon) [reply docs/page/main-features.md]: I agree. How would I go about doing that? The other `Extensions: ` target separate projects with different github.io pages and I am unfamiliar with `mkdocs`. Could you point me in the right direction?

Comment (ericdallo) [reply docs/page/main-features.md]: You can add a entry like this:
```yaml
    - Extensions:
        - UI: https://emacs-lsp.github.io/lsp-ui
        - Treemacs: https://github.com/emacs-lsp/lsp-treemacs
        - Helm: https://github.com/emacs-lsp/helm-lsp
        - Ivy: https://github.com/emacs-lsp/lsp-ivy
        - Iedit: page/integrations/iedit.md
```
Also, I realized we are missing avy integration too 😅 or mentioning in other hidden place

**Review by ericdallo — APPROVED** [review]

File: docs/page/main-features.md (hunk 1 above)
Comment (nbfalcon) [reply docs/page/main-features.md]: Thank you, I've implemented your suggestion. However, I still don't have a way to preview them, as `make local-webpage` fails:
```
[...]
docker run --rm -it -p 8000:8000 -v /home/nikita/Projects/emacs-package-contrib/lsp-mode:/docs docker.pkg.github.com/emacs-lsp/docs-image/docs-image
Unable to find image 'docker.pkg.github.com/emacs-lsp/docs-image/docs-image:latest' locally
docker: Error response from daemon: unauthorized: Your request could not be authenticated by the GitHub Packages service. Please ensure your access token is valid and has the appropriate scopes configured.
See 'docker run --help'.
make: *** [Makefile:99: local-webpage] Error 125
```

Do you know what might be wrong? Why would it need to authenticate itself with github? My system is Ubuntu 20.04 LTS, I have docker installed and my user is part of the `docker` group.

**Comment by yyoncho** [comment]
Can you resolve the conflicts?

**Comment by nbfalcon** [comment]
I've rebased this against `master`.

**Review by yyoncho — APPROVED** [review]

**Comment by yyoncho** [comment]
Thank you!

**Pull request merged**